store/metadata.go

```diff
@@ -361,15 +384,30 @@ func (ms *MetadataStore) Reload() error {
 
 	hostsMap := getHostsMapFromHostsArray(hosts)
 
+	networks, err := ms.mc.GetNetworks()
+	if err != nil {
+		logrus.Errorf("couldn't get networks from metadata: %v", err)
+		return err
+	}
+	networksMap := getNetworksMapFromNetworksArray(networks)
+
+	selfNetwork, ok := networksMap[selfContainer.NetworkUUID]
+	if !ok {
+		logrus.Errorf("couldn't find selfNetwork: %v", err)
```

Comment:
Minor: we probably already log this error in the caller stack, so no need to logrus.errorf here